soh/include/z64effect.h

```diff
@@ -107,11 +120,12 @@ typedef struct {
     /* 0x19E */ u8 numElements; // "now_edge_num"
     /* 0x19F */ u8 elemDuration;
     /* 0x1A0 */ u8 unkFlag;
-    /* 0x1A1 */ u8 drawMode; // 0: simple; 1: simple with alt colors; 2+: smooth
-    /* 0x1A2 */ Color_RGBA8 altPrimColor; // used with drawMode 1
-    /* 0x1A6 */ Color_RGBA8 altEnvColor; // used with drawMode 1
-    /* 0x1A0 */ u8 trailType; // 1 is default swords, 2 is boomerang, 3 is bombchu, 0 is rest. 4 is Kokiri, 5 is Master, 6 is BGS, 7 is Stick, 8 is Hammer.
-} EffectBlure; // size = 0x1AC
+    /* 0x1A1 */ u8 drawMode;                    // 0: simple; 1: simple with alt colors; 2+: smooth
+    /* 0x1A2 */ Color_RGBA8 altPrimColor;       // used with drawMode 1
+    /* 0x1A6 */ Color_RGBA8 altEnvColor;        // used with drawMode 1
+    /* 0x1A0 */ TrailType trailType; // 1 is default swords, 2 is boomerang, 3 is bombchu, 0 is rest. 4 is
```

Comment:
Not that you added these but would it be possible to get rid of the commented offsets here for the stuff SoH has added? Mostly saying this because these offsets are wrong, but it helps make clear that this is not part of decomp (since it doesn't necessarily look that way)

Reply:
Right, I was unsure about where the decomp started and ended here. Should I just remove the offsets on the four lines you commented on or everything in this struct?

Reply:
it would be just these four lines for this struct. but other structs with this enum are also wrong. you can compare with the original decomp but dont worry if not